Gruntfile.js

```diff
@@ -1247,6 +1247,8 @@ module.exports = function(grunt) {
 		 */
 		grunt.task.run( 'wp-packages:refresh-deps' );
 
+		grunt.task.run( 'build' );
```

Comment:
https://github.com/WordPress/wordpress-develop/blob/94e4dfdb235f26d774e654e66cfe708e48028631/package.json#L158

https://github.com/WordPress/wordpress-develop/blob/94e4dfdb235f26d774e654e66cfe708e48028631/package.json#L159

I think the issue was that we had a wrong grunt task listed. It should be `build`  like in the new commit, but we might also need to pass `--dev` flag.